specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzeconversations-authoring.json

```diff
@@ -1402,6 +1998,17 @@
           "items": {
             "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedPrebuiltEntity"
           }
+        },
+        "regex": {
+          "description": "The regex component of the entity.",
```

Comment:
Can you describe what regex syntax is required e.g., .NET, Perl, etc? There are so many dialects that differ, I think it's important say at least what dialect is supported here, ideally with a link.

Reply:
Will ask the developer who owns regex for clarification on this.